recipes/gdk-pixbuf/all/test_v1_package/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package C)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(gdk-pixbuf REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.c)
```

You can avoid duplicating the `test_package.c` file like this:

```suggestion
add_executable(${PROJECT_NAME} ../test_package/test_package.c)
```